web/blog/2022-01-27-waspleau.md

```diff
@@ -19,6 +17,10 @@ import ImgWithCaption from './components/ImgWithCaption'
   <Link to={'https://waspleau.netlify.app/'}>See Waspleau here!</Link> | <Link to={'https://github.com/wasp-lang/wasp/blob/main/examples/waspleau'}>See the code</Link>
 </p>
 
+We've built a Tableau clone using Wasp!
```

Comment:
@shayneczyzewski Better ideas for the summary are welcome (the point of summaries is explained [here](todo linkaj))!

Reply:
Thanks for giving it a summary, @sodic. In this case since it is super simple, I'd maybe go: `We've built a dashboard powered by a job queue using Wasp!` instead? Thanks!